e2e/tests/submissions/author.test.js

```diff
@@ -0,0 +1,39 @@
+const { ResponseInterceptor } = require('../../utils/interceptors');
+const { login } = require('../../utils/user');
+const { submitForm, waitForSubmissionSuccess } = require('../../utils/dom');
+const routes = require('../../utils/routes');
+
+describe('author submissions', () => {
+  it('submits new author', async () => {
+    const context = await browser.createIncognitoBrowserContext();
+    await login(context);
+
+    const page = await context.newPage();
+    await page.goto(routes.AUTHOR_SUBMISSION);
+
+    const interceptor = new ResponseInterceptor(page);
+
+    await submitForm(page, {
+      given_name: 'Diego',
+      family_name: 'Martínez Santos',
+      display_name: 'Diego Martínez',
+    });
+    await waitForSubmissionSuccess(page);
+
+    const submitResponse = interceptor.getFirstResponseByUrl(
+      routes.AUTHOR_SUBMISSION_API
+    );
+    const submitResponseJson = await submitResponse.json();
+    const workflowResponse = await page.goto(
+      `${routes.HOLDINGPEN_API}/${submitResponseJson.workflow_object_id}`
+    );
+    const workflowJson = await workflowResponse.json();
+
+    expect(workflowJson.metadata.name).toEqual({
+      preferred_name: 'Diego Martínez',
+      value: 'Martínez Santos, Diego',
+    });
+    expect(workflowJson.metadata._collections).toEqual(['Authors']);
```

Comment:
we have to assert more:

* ``metadata.acquisition_source.email``
* ``metadata.acquisition_source.method``
* ``_workflow.workflow_name``